docs/framework/develop-client-apps.md

```diff
@@ -27,7 +27,7 @@ ms.author: dotnetcontent
 manager: "wpickett"
 ---
 # Developing Client Applications with the .NET Framework
-There are multiple ways to develop Windows-based applications with the .NET framework that run locally on users' computers or devices. This section contains topics that describe how to create Windows-based applications by using Windows Presentation Foundation (WPF) or by using Windows Forms. However, you also create client applications using the .NET Framework and make them available to your users through the Windows Store and Windows Phone store for computers or devices, web applications that can be viewed in a browser.  
+There are multiple ways to develop Windows-based applications with the .NET framework that run locally on users' computers or devices. This section contains topics that describe how to create Windows-based applications by using Windows Presentation Foundation (WPF) or by using Windows Forms. However, you can also create client applications using the .NET Framework and make them available to your users through the Windows Store and Windows Phone store for computers or devices, and web applications that can be viewed in a browser.  
```

Comment:
for computers or devices, and web applications --> for computers, devices, and web applications

Reply:
Thanks Ron. The only thing is that makes it sounds like web applications are available through the Windows Store and Windows Phone Store. How about:

However, you can also create web applications using the .NET Framework, and client applications  for computers or devices that you make available through the Windows Store or Windows Phone Store.

Reply:
That sounds good, @gewarren.